doc/howto/build-linux.md

```diff
@@ -0,0 +1,105 @@
+## Build and configure X-Ray 1.6 on Linux
+
+Make sure you are using STALKER Call Of Pripyat v1.6.02 and your system contains the following packages:
+git, cmake, GLEW, Freeimage, Lockfile, Openal, TBB, Crypto++, pugixml, Theora, Ogg, SDL2, Lzo, Jpeg.
+
+To install software above in **Debian** and **Ubuntu** derivatives just run the command:
+
+```
+sudo apt install git cmake libglew-dev libfreeimage-dev liblockfile-dev libopenal-dev libtbb-dev libcrypto ++ - dev libpugixml-dev libogg-dev libtheora-dev libvorbis-dev libsdl2-dev liblzo-dev-libjdg2-dev
+```
+
+In case you are **Fedora** user:
+```
+sudo dnf install git cmake glew-devel freeimage-devel liblockfile-devel openal-devel tbb-devel cryptopp-devel pugixml-devel libogg-devel libtheora-devel libvorbis-devel SDL2-devel lzo-devel libjpeg-turbo-devel
+```
+Next clone the repo with its submodules:
+```
+git clone https://github.com/OpenXRay/xray-16.git && git submodule update --init --recursive
```

Comment:
Lil' bit wrong - to download submodules you'll have to cd to cloned xray-16 folder first, or else you'll get error
Should be like that:
git clone https://github.com/OpenXRay/xray-16.git && **cd xray-16 &&** git submodule update --init --recursive

Reply:
```suggestion
git clone https://github.com/OpenXRay/xray-16.git --recursive
```